src/components/BlockInfo.vue

```diff
@@ -169,4 +199,14 @@ export default {
 code::before {
   content: '';
 }
+h3.block-info {
+  position: relative;
+}
+  h3.block-info > p {
```

Comment:
Is it Github that is putting a weird tabulation here ?

Reply:
Just an [old habit](https://css-tricks.com/different-ways-to-format-css/#article-header-id-1) with css files. But yes, probably not needed here.